shards/run-test-ipfs.edn

```diff
@@ -47,13 +42,12 @@
    (->
     (Log "connecting to p2p address")
     = .p2p-address
-    ["swarm" "peering" "add" .p2p-address] = .ipfs-args
+    ["swarm" "peering" "add" .p2p-address] >= .ipfs-args
     "" (Process.Run :Executable "ipfs" :Arguments .ipfs-args)
-    )
-   )
+    ["swarm" "connect" .p2p-address] > .ipfs-args
```

Comment:
Was this added on devel in the end @alebaffa ? If so close this please 🙏

Reply:
Yes, it was added.